requirements-build.txt

```diff
@@ -4,6 +4,6 @@ ninja
 packaging
 setuptools>=61
 setuptools-scm>=8
-torch==2.5.1
+torch==2.6.0
```

Comment:
Would you be willing to update `requirements-cpu.txt` as well?

On my Mac, `uv pip install "torch>=2.6" "vllm@git+https://github.com/vllm-project/vllm.git@update-torch-2.6.0"` doesn't quite work yet due to this

Reply:
The PR description says that other devices should be upgraded separately